tests/functional/strategy/test_migration.py

```diff
@@ -25,8 +25,8 @@ def test_good_migration(
         vault.migrateStrategy(strategy, new_strategy, {"from": guardian})
 
     vault.migrateStrategy(strategy, new_strategy, {"from": gov})
-    assert vault.strategies(strategy)[4] == 0
-    assert vault.strategies(new_strategy)[4] == strategy_debt
+    assert vault.strategies(strategy).dict()["totalDebt"] == 0
+    assert vault.strategies(new_strategy).dict()["totalDebt"] == strategy_debt
```

Comment:
Note that these were previously incorrect, so there is a latent failure in this test case